### PR TITLE
Provide some documentation as README

### PR DESCRIPTION
**Pull request by w00lf**
metanorma/metanorma-plugin-datastruct#6 documentation for yaml2text, json2text usage

### CODE REVIEW

**Comment by w00lf** [comment]
ping @ronaldtse

File: README.adoc [hunk 1]
```diff
@@ -4,13 +4,595 @@
...
 [source,console]
 ----
 $ gem install metanorma-plugin-datastruct
 ----
 
-== Documentation
+== Usage
 
-See http://metanorma.com[]
+In order to use macroses(yaml2text, json2text) you need to register them as Asciidoctor extentions in metanorma.
```
Comment (ronaldtse) [inline README.adoc:16]: @w00lf can we write something like:

In order to use the macros in Metanorma, add the gem `gem 'metanorma-plugin-datastruct'` in your Gemfile. Or have the gem installed, and Metanorma can use them automatically via the extension `:plugin: datastruct`.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (w00lf) [reply README.adoc]: Changed

File: README.adoc [hunk 2]
```diff
@@ -4,13 +4,595 @@
...
 
-== Documentation
+== Usage
 
-See http://metanorma.com[]
+In order to use the macros in Metanorma, add the gem gem 'metanorma-plugin-datastruct' in your Gemfile. Or have the gem installed, and Metanorma can use them automatically via the extension :plugin: datastruct.
+
+EXAMPLE:
+
+[source,ruby]
```
Comment (ronaldtse) [inline README.adoc:20]: We want to remove this block because this is internal Metanorma code. Users will never use this.

Comment (w00lf) [reply README.adoc]: done, removed

**Comment by ronaldtse** [comment]
Thanks @w00lf !

**Pull request merged**